library/src/main/kotlin/com/chuckerteam/chucker/internal/support/ExportFormat.kt

```diff
@@ -0,0 +1,6 @@
+package com.chuckerteam.chucker.internal.support
```

Comment:
This is now part of the public API. Should be inside the 
`com.chuckerteam.chucker.support`

You also need to run `apiDump` to regenerate the API files

Reply:
You mean `com.chuckerteam.chucker.api`as RetentionManager?

Reply:
Sorry yes, the one you suggested is the correct one